### PR TITLE
make ForecastScope again, remove ForecastScopeID

### LINKED ISSUE #98: Forecast uses Scope object directly

### PR DESCRIPTION
**Pull request by cwhanse**
Closes #98

### CODE REVIEW

**Review by jrippingale — APPROVED** [review]
I didn't realize we had an external id value also.  Thank you for aligning this!

**Review by jrippingale — CHANGES_REQUESTED** [review]
The description references Forecase (rather than forecast in the whatnew file

File: docs/whatsnew/2207.1.0.rst [hunk 1]
```diff
@@ -22,6 +23,7 @@ Object changes
 * EnergyModel elements changed (:pull:`88`)
 * Adds InsurancePolicy, InsurancePolicyScope (:pull:`103`)
 * Adds element CECListingDate to ProdModule, ProdInverter (:pull:`104`)
+* Remake ForecastScope and attach to Forecase (:pull:`105`)
```
Comment (cwhanse) [inline docs/whatsnew/2207.1.0.rst:26]:
```suggestion
* Remake ForecastScope and attach to Forecast (:pull:`105`)
```

**Review by jrippingale — APPROVED** [review]
Looks correct to me!  Thank you!

**Pull request merged**